Source/WebCore/Modules/indexeddb/IDBTransaction.cpp

```diff
@@ -631,6 +631,11 @@ void IDBTransaction::dispatchEvent(Event& event)
     ASSERT(event.type() == eventNames().completeEvent || event.type() == eventNames().abortEvent);
     m_didDispatchAbortOrCommit = true;
 
+    if (!m_openDBRequest) {
+        ASSERT(m_isStopped);
+        return;
+    }
+
```

Comment:
The assertion seems incorrect; a non-versionchange transaction can have null m_openDBRequest and is non-stopped (transaction created by `static Ref<IDBTransaction> create(IDBDatabase&, const IDBTransactionInfo&)`)
I think this should be moved to `if (isVersionChange()) {`; or we can make early return if it's not versionchange transaction

Reply:
Ah, yes silly mistake. Thank you